docs/README.md

```diff
@@ -1397,14 +1397,33 @@ export class UserReadModel {
   public constructor(readonly username: string, /* ...(other interesting fields from users)... */) {}
   
   @Projects(User, 'id')
-  public static projectUser(entity: User, current?: UserReadModel) { // Here we update the user fields}
+  public static projectUser(entity: User, current?: UserReadModel): ProjectionResult<UserReadModel> { // Here we update the user fields}
 
   @Projects(Post, 'ownerId')
-  public static projectUserPost(entity: Post, current?: UserReadModel) { //Here we can adapt the read model to show specific user information related with the Post entity}
+  public static projectUserPost(entity: Post, current?: UserReadModel): ProjectionResult<UserReadModel> { //Here we can adapt the read model to show specific user information related with the Post entity}
 }
 ```
 In the previous example we are projecting the `User` entity using the user `id` and also we are projecting the `User` entity based on the `ownerId` of the `Post` entity. Notice that both join keys are references to the `User` identifier, but it's not required that the join key is an identifier.
 
+You can also delete read models by returning the `deleteReadModel` type, as shown in the following example:
+```
+@ReadModel
+export class UserReadModel {
+  public constructor(readonly username: string, /* ...(other interesting fields from users)... */) {}
+  
+  @Projects(User, 'id')
+  public static projectUser(entity: User, current?: UserReadModel): ProjectionResult<UserReadModel> {
+    if (isReadModelDeletable()) {
+        return deleteReadModel 
+    }
+    return new UserReadModel(...)
+  }
+
+  private isReadModelDeletable(): boolean {
+    ...
+  }
```

Comment:
I'd try to make a less generic example, like adding a field called `deleted` or something, and then checking `if (current?.deleted)` or something